module-db/Tables/MultimediaFilesTable.cpp

```diff
@@ -330,4 +330,13 @@ namespace db::multimedia_files
 
         return (*queryRet)[0].getUInt32();
     }
+
+    auto MultimediaFilesTable::getLimitOffsetByPath(const std::string &path, uint32_t offset, uint32_t limit)
+        -> std::vector<TableRow>
+    {
+        std::string query = "SELECT * FROM files WHERE path LIKE '" + path + "%%' ORDER BY title ASC LIMIT " +
```

Comment:
will it work in all cases? e.g. what if path will be music1/ and music2?

Reply:
Added some UT for this. It seems to work, however those are not very detailed and thought through.